test/e2e/cloud-provider-azure.go

```diff
@@ -50,12 +52,20 @@ func InstallCalicoAndCloudProviderAzureHelmChart(ctx context.Context, input clus
 	}
 	// If testing a CI version of Kubernetes, use CCM and CNM images built from source.
 	if useCIArtifacts {
-		options.Values = append(options.Values, fmt.Sprintf("cloudControllerManager.imageName=%s", os.Getenv("CCM_IMAGE_NAME")))
-		options.Values = append(options.Values, fmt.Sprintf("cloudNodeManager.imageName=%s", os.Getenv("CNM_IMAGE_NAME")))
+		ccmImageName := os.Getenv("CCM_IMAGE_NAME")
```

Comment:
this code is not needed: when `useCIArtifacts` is true (which is always true in this if block), we're calling `ci-build-azure-ccm.sh` in ci-e2e.sh. This means the `CCM_IMAGE_NAME` and `CNM_IMAGE_NAME` image vars will always be set by the time we reach this code.

Reply:
Updated.